src/Microsoft.DotNet.Tools.Compiler.Csc/Program.cs

```diff
@@ -124,9 +125,10 @@ private static IEnumerable<string> GetDefaultOptions()
                 ? "-debug:full"
                 : "-debug:portable");
 
-            args.Add("-nowarn:CS1701");
-            args.Add("-nowarn:CS1702");
-            args.Add("-nowarn:CS1705");
+            foreach (var suppress in DefaultCompilerWarningSuppresses.Map["csc"])
+            {
+                args.Add($"-nowarn:{suppress}");
+            }
```

Comment:
This is how it use to add the `-nowarn` parameters. I don't intend to change this behaviour in this PR. Note: only default warning suppresses are added.